packages/framework/request-handler/src/requestHandlers.ts

```diff
@@ -36,9 +36,24 @@ export type RuntimeRequestHandler = (
 export const rootDataStoreRequestHandler = async (
 	request: IRequest,
 	runtime: IContainerRuntime,
-) => {
+): Promise<IResponse | undefined> => {
 	const requestParser = RequestParser.create(request);
 	const id = requestParser.pathParts[0];
+
+	// back-compat: If getAliasDataStore exits, use it to get the data store. Else, fallback to using getRootDataStore.
```

Comment:
Consider omitting this (and we'll just remove this helper entirely when request is removed).  It's not truly backward compatible since you're not satisfying the "wait" flag, and either way we're going to be giving guidance that folks should prefer calling the new API directly.

Reply:
That makes sense. I was mainly trying this out to see if our tests pass and they do (there is one failing which I am debugging right now).

Reply:
done.